pkg/vcr/registry.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2023 OnMetal authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vcr
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	ori "github.com/onmetal/onmetal-api/ori/apis/volume/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func LoadVolumeClasses(reader io.Reader) ([]ori.VolumeClass, error) {
+	classList := []ori.VolumeClass{}
+	if err := yaml.NewYAMLOrJSONDecoder(reader, 100).Decode(&classList); err != nil {
```

Comment:
We should have a more sensible default (`4096`) here - see e.g. `controller-utils/unstructuredutils`